DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java

```diff
@@ -67,8 +69,8 @@ public AlarmViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(AlarmViewHolder holder, final int position) {
-        Alarm alarm = mAlarmList.get(position);
+    public void onBindViewHolder(AlarmViewHolder holder,@SuppressLint("RecyclerView")  final int position) {
```

Comment:
What is the warning being suppressed? (what does it complain about if you remove the `@SuppressLint` annotation?)